sdk/test/trace/simple_processor_test.cc

```diff
@@ -1,67 +1,31 @@
 #include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
```

Comment:
I assume we don't need this include, as it is included by `in_memory_span_exporter.h`.